internal/commands/root.go

```diff
@@ -2,15 +2,14 @@ package commands
 
 import (
 	"fmt"
-	"io/ioutil"
-	"os"
-
 	"github.com/docker/app/internal"
 	"github.com/docker/app/internal/store"
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/cli/config"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+	"io/ioutil"
+	"os"
```

Comment:
Feels like your goimport/gomft is not configured 😅

Reply:
Yes you're right 😞